frontend/src/components/individuals/IndividualsDetailContent.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { Tabs } from 'antd'
+import useHashURL from '../../hooks/useHashURL'
+import AppPageHeader from "../AppPageHeader";
+import PageContent from "../PageContent";
+import EditButton from "../EditButton";
+import IndividualOverview from "./IndividualOverview";
+import { useAppDispatch, useAppSelector } from "../../hooks";
+import { selectIndividualsDetailsById } from "../../selectors";
+import IndividualAssociatedSamples from "./IndividualAssociatedSamples";
+import { listTable } from "../../modules/individualDetails/actions";
+import { Individual, IndividualDetailsState as IndividualsDetailsById } from "../../modules/individualDetails/reducers";
+import { FetchedState } from "../../modules/common";
+import { Sample } from "../../models/frontend_models";
+import { SampleAndLibrary } from "../shared/WorkflowSamplesTable/ColumnSets";
+
+const IndividualsDetailContent = () => {
+    const { id } = useParams();
+    const dispatch = useAppDispatch()
+    const individualDetailsById: IndividualsDetailsById = useAppSelector(selectIndividualsDetailsById)
+
+    const [activeKey, setActiveKey] = useHashURL('overview')
+    const [individual, setIndividual] = useState<FetchedState<Individual>>()
+
+    useEffect(() => {
+        dispatch(listTable(Number(id)));
+    }, [id, dispatch])
+
+    useEffect(() => {
+        if (individualDetailsById[Number(id)]) {
+            const individualInstance: FetchedState<Individual> = {
+                ...individualDetailsById[Number(id)]
+            }
+            if (individualInstance) {
+                setIndividual(individualInstance)
+            }
+        }
+    }, [individualDetailsById])
+
+    const tabsStyle = {
```

Comment:
Nafiz changed some CSS and we don't need to set the `tabsStyle` and `tabPanelStyle` styles on our Tab and TabPane components anymore, so you can trash this.